### PR TITLE
Add support for UUID and BigDecimal

### PR DESCRIPTION
**Pull request by BritishPearl**
~Stacked on the top of #31. Only the most recent commit is actually related to UUID/BigDecimal.~

### CODE REVIEW

File: src/main/scala-2.10/com/mpc/scalats/core/ScalaParser.scala [hunk 1]
```diff
@@ -101,14 +104,24 @@ final class ScalaParser(logger: Logger) {
...
-        tpe.typeSymbol.name.toString,
+        buildTypeName(tpe.typeSymbol),
         ListSet.empty ++ members,
         parseTypes(possibilities)))
 
       case _ => Option.empty[SealedUnion]
     }
   }
 
+  private def parseEnumeration(enumerationValueType: Type): Option[Enumeration] = {
```
Comment (cchantep) [inline src/main/scala-2.10/com/mpc/scalats/core/ScalaParser.scala]: that should be only in the other PR

Comment (BritishPearl) [reply src/main/scala-2.10/com/mpc/scalats/core/ScalaParser.scala]: Done, now this PR is based directly on master

**Comment by BritishPearl** [comment]
@cchantep rebased directly onto master

**Comment by BritishPearl** [comment]
@cchantep merge? I can't do it unfortunately myself :/

**Pull request merged**